tests/integration/test_main.py

```diff
@@ -0,0 +1,179 @@
+# """
```

Comment:
Why are all the tests commented out?

Reply:
I did not manage to make them work on time for the release.